packages/experimental/Button/src/Button.types.ts

```diff
@@ -46,6 +47,9 @@ export interface ButtonCoreTokens extends LayoutTokens, FontTokens, IBorderToken
    */
   spacingIconContent?: number;
 
+  rightIcon?: ButtonTokens;
```

Comment:
I don't see these being used, could you explain the intent of adding these tokens?

Reply:
I thought I removed it. Thank you for catching it